topostats/default_config.yaml

```diff
@@ -51,6 +51,7 @@ grainstats:
   cropped_size: 40.0 # Length (in nm) of square cropped images (can take -1 for grain-sized box)
 dnatracing:
   run: true # Options : true, false
+  save_trace_json_data: false # Whether to save things such as height traces to JSON files.
```

Comment:
Would you be able to add an equivalent row into `docs/configuration.md` for this new option please? :pray: